### DISCUSSION TITLE
Ordering PCB: solder mask openings flagged by JLC

### DISCUSSION THREAD

**Discussion by last5bits** [JellyTitan/Sofle-Pico#38] [General]
First, thank you very much for your hard work on this keyboard! I'm a total newbie and this is my first time ordering a PCB, so any help would be much appreciated.

After submitting the 3.5.4 version of the keyboard to the PCB vendor, they asked me this:

> As shown below, there's no solder mask opening for below indicated holes. Here we have 2 suggestions. Please make a choice.
>
> A: Open them to be covered by tin on both sides. So that they can be soldered components.
>
> B:Follow the file to cover them with solder mask ink.

![images](https://github.com/user-attachments/assets/74fe9b08-0824-430c-96eb-52639b41add8)

![images1](https://github.com/user-attachments/assets/ea870bf1-348b-48d0-ad17-eefa864c644e)

Which one is it?

**Comment by JellyTitan** [comment]
Some of the solder masks have been intentionally omitted so that components can't be soldered in the wrong orientation. 

I haven't seen this vendor error before though - and those arrows are all over the place, which makes it hard to tell exactly what they are getting at.
I would recommend going with option A.
It sounds like they will tweak the file to allow for soldering on the unsolder-able areas - which should be fine. 
During assembly, be sure to pay close attention to which side the components are mounting on.

Which manufacturer is this, btw?

**Comment by last5bits** [comment]
Thanks so much! This is JLCPCB.

**Comment by JellyTitan** [comment]
@last5bits when those PCB's come in, can you post some pictures?

**Comment by last5bits** [comment]
Sure thing

![IMG_20240802_115928_995](https://github.com/user-attachments/assets/97c609df-ade2-4bbf-815f-9eb91b6b74f3)

![IMG_20240802_115936_894](https://github.com/user-attachments/assets/2d66a1a8-f16e-4042-a4fd-ade23e7e9f74)

**Comment by JellyTitan** [comment]
Oh! Thats really interesting - they essentially de-poka-yoke'd it by adding solder masks to both sides. 
I see how that could be perceived as an error.
Thanks for posting - i'll update the help section & the ordering section with a note about JLC flagging it. 

I wonder if "Confirm Production File" kicked off this issue - do you recall if you checked that option?

![image](https://github.com/user-attachments/assets/29d021c4-9b0c-4f74-bd66-a6c7da48c7b2)

**Comment by last5bits** [comment]
No, I chose "No" for "Confirm Production file".
